code/game/machinery/launch_pad.dm

```diff
@@ -235,7 +235,6 @@
     src.briefcase = briefcase
 
 /obj/machinery/launchpad/briefcase/Destroy()
-	QDEL_NULL(briefcase)
```

Comment:
?

Reply:
both the briefcase and the launchpad both call QDEL on each other but the launchpad is inside the briefcase so there is not really much reason to call QDEL again trough i did forget to null the reference

Reply:
Trough maybe it makes more sense to just check for QDELETED just to be sure

Reply:
actually now thinking about it why not use the macro QDELETED directly inside the QDEL_NULL macro and make it conditional that would be one way to ensure this won't happen